scripts/release_info.py

```diff
@@ -292,24 +303,111 @@ def _check_storage_models(current_release):
     return [item for item in diff_list if item.startswith('core/storage')]
 
 
+def check_blocking_bugs(repo):
+    """Checks the number of unresolved blocking bugs.
+
+    Args:
+        repo: github.Repository.Repository. The PyGithub object for the repo.
+
+    Returns:
+        int: Number of unresolved blocking bugs.
+    """
+    blocking_bugs_milestone = repo.get_milestone(number=39)
+    return blocking_bugs_milestone.open_issues
+
+
+def check_prs_for_current_release(repo):
+    """Checks that all pull requests for current release have a
+    PR: released label.
+
+    Args:
+        repo: github.Repository.Repository. The PyGithub object for the repo.
+
+    Returns:
+        int. Whether all pull requests for current release have a
+            PR: released label.
+    """
+    all_prs = repo.get_pulls(state='all')
```

Comment:
will it fetch all the PRs? (If this will be expensive then we can create a separate milestone for release related PRs.)

Reply:
I tried this and it was not time-expensive. It works fine. You can try it once to check if it works fine and let me know. I will make the updates accordingly.

Reply:
On further thoughts, I feel that this may become expensive as the number of PRs keeps increasing. I am not sure if adding a milestone for release related PRs will help us to tackle this because we can only obtain the number of open/closed PRs in the milestone not the details of the PR. So, should we instead make a get request instead and fetch PRs with both the labels and compare the output?

Reply:
If you can only fetch the PRs with the "for release" label, that might be better. Then you can iterate along them and ensure that they all have the "released" label as well.

But if that is not feasible, then fetch all PRs for safety. If you add a milestone then it becomes confusing, since we use milestones to track tasks (issues) and not PRs. Plus, this is susceptible to human error.

Also, I'm a bit confused about one thing -- you mentioned that there's no way in pygithub to filter the PRs, but you also suggested making a get request to fetch PRs with both the labels (which seems like a sort of filtering). So ... what is the exact situation here?

Reply:
I did not find any function in pygithub to fetch pull requests using a filter but the solution on stackoverflow [here](https://stackoverflow.com/a/36984550) provides a way to fetch using a get request through github api.

Reply:
Ah ok, thanks for clarifying.